src/Sulu/Bundle/AudienceTargetingBundle/SuluAudienceTargetingBundle.php

```diff
@@ -40,6 +44,15 @@ public function build(ContainerBuilder $container)
             $container
         );
 
+        $container->addAliases(
```

Comment:
not sure if it is bad practice to do this in the `Bundle` class. but it would be nice to do it here because we also need to call the `buildPersistence` method with all model interfaces 🙂

Reply:
The `Bundle` is where no services should be in my opinion defined. The Bundle is place to define CompilerPasses. Service should in our case be defined in the `xml` aslong as we use `xml` for service definitions, some aliases maybe in the `Extensions:load` which need to be programmitically, Compilerpasses mostly should only in edge cases define new services.

Reply:
alright - thanks for the explanation. i moved the logic into the `Extension` classes 🙂

Reply:
I agree with you that we should use the `xml` files for defining services in general. But i think in this case it would be nice to do it in the `Extension` class after the `configurePersistence` call. This makes it easier to understand that the `sulu.repository.*` services are created by the `PersistenceBundle`. Also we cannot forget about creating the alias when copying the code from another bundle this way 😉 

But I dont have a too strong opinion here. The current solution works for me too, just think this would make things easier to understand 🙂

Reply:
is okay for if errors could prevent with this in future we could use in this case php